app/utils/emoji/helpers.ts

```diff
@@ -370,3 +369,24 @@ export const searchEmojis = (fuse: Fuse<string>, searchTerm: string) => {
 
     return [];
 };
+
+export const getEmojiCode = (emoji: string, customEmojis: CustomEmojiModel[] = [], ios = false) => {
```

Comment:
the last argument can be removed and use the `Platform` module to identify if is iOS